playbooks/openshift-master/private/openshift_node_config.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Configure node configmap on the master
+  hosts: oo_first_master
+  gather_facts: no
+  tasks:
+  - name: include openshift node config
+    import_tasks: tasks/node_config.yml
```

Comment:
No need for this extra layer of imports if it's only one task; just import_role in both places.

Reply:
fixed, thanks for the direction.